Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions.tsx

```diff
@@ -139,9 +139,16 @@ const SubmissionsProvider = ({ children }: ISubmissionsProviderProps) => {
         [ currentProblem ],
     );
 
-    const updateSubmissionCode = (code: string | Blob) => {
-        setSubmissionCode(code);
-    };
+    const updateSubmissionCode = useCallback(
+        (code: string | Blob) => {
+            setSubmissionCode(code);
+
+            if (!isNil(currentProblem)) {
+                currentProblem.codeEditorCode = code;
```

Comment:
destruct

Reply:
I just want to assign the value of code to the property codeEditorCode of currentProblem and if I try to destructure it and then assign it the value of code, it says the value of codeEditorCode is not used anywhere.

Reply:
This is not correct because if the code is a blob we dont have codeEditorCode at all